lambda/marchon.py

```diff
@@ -76,32 +94,62 @@ def get_sheet_data():
      16 Event Link, 17 Photo, 18 About
     '''
     # keep these fields
-    fields = {'name': 0, 'location': 1, 'contactName': 3, 'contactEmail': 5,
-              'facebook': 9, 'twitter': 10, 'instagram': 11, 'other': 12,
-              'website': 13, 'event': 14, 'eventDate': 15, 'eventLink': 16,
-              'photo': 17, 'about': 18}
+    fields = {
+        'name': 0,
+        'location': 1,
+        'contactName': 3,
+        'contactEmail': 5,
+        'facebook': 9,
+        'twitter': 10,
+        'instagram': 11,
+        'other': 12,
+        'website': 13,
+        'event': 14,
+        'eventDate': 15,
+        'eventLink': 16,
+        'photo': 17,
+        'about': 18
+    }
     return read_sheet('Sheet1!A1:S', fields, 1, True)
 
 
+def get_action_network_events() -> Dict:
```

Comment:
why make a function that only calls a function? call the function directly instead